app.py

```diff
@@ -86,12 +106,95 @@ def delete(id):
         return redirect(url_for('history'))
 
 
+#Database API for getting charts
+@app.route('/request/chart/<id>', methods=['POST', 'GET'])
+def updateChart(id):
+    getDate = records.query.order_by()
+    toAppend = []
+    for n in getDate:
+        toAppend.append(n.time)
+    sortedArray = sorted(toAppend, key=sortingDates)
+    #function to add up all records by id
+    sumArray = []
+    numCache = []
```

Comment:
Auch diese Variablen Namen sind sehr allgemein gehalten. Es macht mehr Sinn, sie nach den Daten zu benennen, die darin gespeichert sind. z.B. listOfIntermediateRecordSum (wenn es das ist :smile: ) Mit den allgemeinden Namen fängt man in 6 Monaten nicht mehr so viel an und muss erstmal wieder anfangen zu überlegen, was hier gemacht wird.